csrankings.csv

```diff
@@ -13744,6 +13744,7 @@ Nahid Farhady Ghalaty,Texas Tech University,http://www.depts.ttu.edu/cs/departme
 Nahri Moreano,UFMS,https://www.facom.ufms.br/~nahri,NOSCHOLARPAGE
 Nahum Shimkin,Technion,http://webee.technion.ac.il/Sites/People/shimkin,YPptH48AAAAJ
 Naijie Gu,USTC,http://dsxt.ustc.edu.cn/zj_ywjs.asp?zzid=745,NOSCHOLARPAGE
+Naijun Zhan,Chinese Academy of Sciences,http://lcs.ios.ac.cn/~znj,bd-XZkYAAAAJ

```

Comment:
website says "research professor"; only tenure-track faculty are currently eligible for inclusion.

Reply:
Hi Emery,

   Prof. Naijun Zhan is also a tenure-track faculty. I update his website in UCAS (University of Chinese Academy of Sciences) to reflect this.
   Thanks.

Reply:
But, his website in UCAS is written in Chinese. If possible, http://lcs.ios.ac.cn/~znj could be better. 
Thanks.